dll/cpl/input/lang/de-DE.rc

```diff
@@ -1,14 +1,26 @@
+/*
+ * PROJECT:     ReactOS Language Control Panel
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     German resource file
+ * TRANSLATORS: Copyright 2007-2014 Daniel Reimer <reimer.daniel@freenet.de>
+ *              Copyright 2008 Colin Finck <colin@reactos.org>
+ *              Copyright 2008-2011 Matthias Kupfer <mkupfer@reactos.org>
+ *              Copyright 2010 Sven Knurr (tux.) <forenkrams@tuxproject.de>
+ *              Copyright 2018 Ștefan Fulea <stefan.fulea@mail.com>
```

Comment:
I am not sure Here.

Reply:
I took a look at the commit history.
The change that Stefan did was not regarding translation, but adding additional information (in all other language files too)
So please remove him as translator.